docs/what_is_data_science/index.md

```diff
@@ -1,3 +1,23 @@
 # What is Data Science?
 
-> A page describing what data science is.
\ No newline at end of file
+Quoting the [DDaT Capability Framework](https://ddat-capability-framework.service.gov.uk/data-scientist.html):
+
+> Data science is a broad and fast-moving field spanning maths, statistics, software engineering and communications. Data scientists will often work as part of a multidisciplinary team, using data and analytics to inform and achieve organisational goals. 
+
+Specifically, this means that often data scientists use programming languages like Python and R (though there are others too!) to solve problems within the business, or inform about courses of action, working with other colleagues such as analysts and data engineers, in particular. 
+
+Some examples of the kinds of problems data science can help with are:
+
+- **getting more value out of unstructured data** (e.g. text, images, audio) through "Natural Language Processing", Neural Networks and recently "Large Language Models" (e.g. ChatGPT).
+- **modelling systems and forecasting** (though compared with statisticians and economists, this tends to be more empirical, requiring more focus on evaluation)
+- **explaining existing models and their performance** - communication is a key pillar of data science: making the complex understandable to everyone else.
+- **enriching and transforming data**, such as through linkage, feature engineering, artificial and synthetic data generation
+- **classification and regression** - that is saying what something is, or if something will or won't happen, and/or quantifying something unknown.
+
+**Artificial Intelligence** and **Machine Learning** are techniques which are widely associated with Data Science and Data Scientists, and which can be applied to basically any of the problems listed above. See also this article on the ["Seven Patterns of AI"](https://www.forbes.com/sites/cognitiveworld/2019/09/17/the-seven-patterns-of-ai/?sh=448cf51812d0).
+
+
+??? info "How does Data Scientists differ from Analysts and Data engineers?"
+    Data Science can be quite hard to pin down, as it covers a lot of different techniques, and problems, and data scientists themselves often have a lot of overlap with Analysts and Data Engineers. Analysts and Engineers might well use data science techniques in their work!
+
+    However, generally, data scientists are slightly more focussed on looking ahead, embracing and exploiting new techniques, e.g. unstructured data, such as text, images, audio.
```

Comment:
not relevant as a bit of review but just letting you know that youve inadvertedly blown my mind with the fact that "focussed" is actually a correct spelling and it isnt only spelled with one s